src/java.base/share/classes/sun/util/calendar/ZoneInfoFile.java

```diff
@@ -726,8 +737,9 @@ private static ZoneInfo getZoneInfo(String zoneId,
                 checksum = (int)sum.getValue();
             }
         }
+
         return new ZoneInfo(zoneId, rawOffset, dstSavings, checksum, transitions,
-                            offsets, params, willGMTOffsetChange);
+                offsets, params, willGMTOffsetChange);
```

Comment:
do we need to change it here?

Reply:
No need.  It was reformatted by IntelliJ editor.  The original code is better from perspective of the OpenJDK's standard code style.